docs/modules/storage/pages/configuring-persistence.adoc

```diff
@@ -1024,6 +1024,14 @@ include::ROOT:example$/storage/SampleEncryptionAtRestConfiguration.java[tag=vaul
 
 Use the following options to configure persistence for xref:data-structures:map.adoc[map] and xref:jcache:overview.adoc[JCache] data structures.
 
+[NOTE]
+====
+To avoid exception errors, make sure that your map names do not:
+
+* Exceed 255 characters. 
+* Contain special characters (`*``:``"``'``|``<````,``>``?``/`).
```

Comment:
```suggestion
* Special characters (`*``:``"``'``|``<````,``>``?``/`)
```

No need for periods here because these bullets aren't complete sentences.